documentation/whatsnew/openshift/openshift-news-3.0.0.Beta2.adoc

```diff
@@ -0,0 +1,58 @@
+= OpenShift What's New in OpenShift 3.0.0.Beta2
+:page-layout: whatsnew
+:page-component_id: openshift
+:page-component_version: 3.0.0.Beta2
+:page-product_id: jbt_core
+:page-product_version: 4.3.0.Beta2
+
+== Manage your OpenShift Projects
+Starting with JBoss Tools 4.3.0.Beta2 you can now create and delete OpenShift projects. 
+
+image::./images/manage-projects-wizard.png[]
+
+You launch this new wizard from the context menu of connections in the OpenShift Explorer.
+
+image::./images/manage-projects.png[]
+
+If you try to create a new application, but you have no project yet, the tools will prompt you create one first.
+Once you are done you can always get back and manage your OpenShift projects via a link in the application wizard.
+
+image::./images/manage-projects-link.png[] 
+
+related_jira::JBIDE-20016[]
+
+== Manually Trigger Builds
+You can now manually trigger builds when selecting your Build Configs in the OpenShift Explorer.
+
+image::./images/start-build.png[]
+
+Once you triggered you should see a new build appear in the Builds category in the OpenShift Explorer. 
+You can see it's state next to its name or in the Properties view. 
+Refreshing the Explorer will show you when the build completes.
+
+related_jira::JBIDE-20047[]
+
+== Port Forwarding
```

Comment:
if it's not working, we shouldn't mention it. 
if it's partially working, we should say how. add all the expected disclaimers